sdk/src/test/java/com/horizen/vrf/VRFProofTest.java

```diff
@@ -0,0 +1,45 @@
+package com.horizen.vrf;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class VRFProofTest {
+
+    static byte[] bhash = new byte[VRFProof.PROOF_LENGTH];;
+
+    @BeforeClass
+    public static void before() {
+        System.loadLibrary("vrflib");
+        for (byte i = 0; i < bhash.length; i++) bhash[i] = i;
```

Comment:
Let me suggest to store `bhash` and `br` values as some predefined constants (can be even in hex form), so, later when we will have a real rust implementation we'll need just to change the these constants.